webui/portal/src/main/java/org/exoplatform/portal/webui/login/UIForgetPassword.java

```diff
@@ -126,8 +127,7 @@ public void execute(Event<UIForgetPassword> event) throws Exception {
 
             // Create token
             RemindPasswordTokenService tokenService = uiForm.getApplicationComponent(RemindPasswordTokenService.class);
-            Credentials credentials = new Credentials(user.getUserName(), "");
-            tokenId = tokenService.createToken(credentials,tokenService.FORGOT_PASSWORD_TOKEN);
+            tokenId = tokenService.createToken(user.getUserName(), CookieTokenService.FORGOT_PASSWORD_TOKEN);
```

Comment:
This class isn't used anymore, shouldn't we delete it ?

Reply:
Do you mean in import ? 
If yes, I remove it.

Reply:
I removed the complete 'login' folder